prov/rxd/src/rxd_cq.c

```diff
@@ -191,7 +191,7 @@ void rxd_cq_report_tx_comp(struct rxd_cq *cq, struct rxd_x_entry *tx_entry)
 static int rxd_ep_recv_data(struct rxd_ep *ep, struct rxd_x_entry *rx_entry,
 			struct rxd_pkt_entry *pkt_entry, size_t size)
 {
-	struct fi_cq_err_entry err_entry;
+	struct fi_cq_err_entry err_entry = {0};
```

Comment:
This has an initialization cost on every function call (at least for a dumb compiler).  Add a memset 0 (or manually clear the unset fields) only when reporting an error.